src/home_robot_hw/home_robot_hw/env/visualizer.py

```diff
@@ -251,7 +268,7 @@ def visualize(
 
     def _init_vis_image(self, goal_name: str):
         # vis_image = np.ones((655, 1165, 3)).astype(np.uint8) * 255
-        vis_image = np.ones((655, 885, 3)).astype(np.uint8) * 255
+        vis_image = np.ones((655, 1500, 3)).astype(np.uint8) * 255
```

Comment:
i guess this was already in there but these hard-coded values could cause trouble too right?